docs/whats_new.md

```
@@ -25,21 +25,21 @@ See the [migration guide](#-migration-guide) for more details and options.
-- [🧠 Context](#-context)
-- [📎 Binding](#-binding)
-- [🔬 Extractors Package](#-extractors-package)
-- [🔄️ Redirect](#-redirect)
-- [🌎 Client package](#-client-package)
-- [🧰 Generic functions](#-generic-functions)
-- [🛠️ Utils](#utils)
-- [🧩 Services](#-services)
-- [📃 Log](#-log)
-- [📦 Storage Interface](#-storage-interface)
-- [🧬 Middlewares](#-middlewares)
+- [App](#-app)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)